Library/Homebrew/formula_creator.rb

```diff
@@ -116,6 +116,8 @@ class #{Formulary.class_s(name)} < Formula
           uses_from_macos "ruby"
         <% elsif mode == :rust %>
           depends_on "rust" => :build
+        <% elsif mode == :swift %>
+          depends_on :xcode => ["11.4", :build] # swift-tools-version:5.2
```

Comment:
This has the potential to become:
```rb
depends_on :swift => ["5.2", :build]
```
if #7276 gets merged 🤞

Reply:
Might be worth holding off on that?